src/common.h

```diff
@@ -39,3 +39,10 @@
 #define IIF_0(t, ...) __VA_ARGS__
 /* run the 1st parameter */
 #define IIF_1(t, ...) t
+
+#if defined(__has_attribute) && __has_attribute(musttail)
```

Comment:
Move these line right after the part of `Alignment macro`. And, add comments for annotation.

Reply:
Macro `MUST_TAIL` should appear before pattern matching macros.

Reply:
Macro `MUST_TAIL` should appear before pattern matching macros.